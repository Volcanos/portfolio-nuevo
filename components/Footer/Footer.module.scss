.footerWrapper {
  align-items: center;
  border-top: 1px solid #e8e8e8;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  margin: 60px auto 0;
  padding: 20px 0;
  width: 100%;
  font-size: 0.8rem;
}
