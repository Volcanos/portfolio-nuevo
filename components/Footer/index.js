import styles from './Footer.module.scss';

const Footer = () => {
    return (
        <div className={styles.footerWrapper}>
            Diseñado y maquetado por Borja Gracia Martín
            { <a href="https://github.com/Volcanos/portfolio" target="_blank">GitHub</a> }
        </div>
    )
}
export default Footer;