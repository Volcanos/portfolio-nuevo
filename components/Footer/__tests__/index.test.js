import React from 'react'
import { mount, render } from 'enzyme';
import 'jest-styled-components'
import Footer from '../index';

describe('Footer', function () {
    it('should render text', function () {
        const text = render(<Footer />).text();
        expect(text).toMatch(/Borja Gracia Mart√≠n/);
    });
    it('should render link', function () {
        const text = render(<Footer />).html();
        expect(text).toMatch(/<a/);
    });
    it('snapshot not change', function () {
        const tree = render(<Footer />);
        expect(tree).toMatchSnapshot();
    });
});