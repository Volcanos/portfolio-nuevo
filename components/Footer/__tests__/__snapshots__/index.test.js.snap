// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot not change 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-top: 1px solid #e8e8e8;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 60px auto 0;
  padding: 20px 0;
  width: 100%;
  font-size: .8rem;
}

<footer
  class="c0"
>
  Diseñado y maquetado por Borja Gracia Martín
  <a
    href="https://github.com/Volcanos/portfolio"
    target="_blank"
  >
    GitHub
  </a>
</footer>
`;
